<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ERegComplete</class>
 <widget class="QWidget" name="ERegComplete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;center&gt;&lt;p&gt;&lt;font color=&quot;green&quot;&gt;&lt;b&gt;Congratulations!&lt;/b&gt; Employee Registration Complete&lt;/font&gt;&lt;/p&gt;&lt;/center&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="dairyNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;center&gt;&lt;p&gt;RD Square Dairy, Lalya ka Bas Jaipur&lt;/p&gt;&lt;/center&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="empLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black

}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;You have got new employee:&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="empNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>401</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;&lt;b&gt;Name: &lt;/b&gt;&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameSLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: blue
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;&lt;center&gt;Rahul Dangi&lt;/center&gt;&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="depLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>401</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;&lt;b&gt;Department: &lt;/b&gt;&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="deptSLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: blue
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;&lt;center&gt;Mangement Department&lt;/center&gt;&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="desgLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>401</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;&lt;b&gt;Designation: &lt;/b&gt;&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="desgSLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: blue
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;&lt;center&gt;Worker&lt;/center&gt;&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {

color: red
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;p&gt;&lt;center&gt;E-Mail is sent to him/her!&lt;/center&gt;&lt;/p&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
